tests/Argu.Tests/Tests.fs

```diff
@@ -326,6 +328,17 @@ module ``Argu Tests Main List`` =
     let ``Parse equals or space assignment with space`` () =
         let result = parser.Parse([|"--flex-equals-assignment"; "../../my-relative-path"; "--dir==foo"|], ignoreMissing = true)
         test <@ result.GetResult Flex_Equals_Assignment = "../../my-relative-path" @>
+        
+    [<Fact>]
+    let ``Parse equal or space assignment with key-value assignment with equals`` () =
+        let clp = [|"--flex-equals-assignment-arity-2"; "foo=bar"|]
```

Comment:
I would rather make it so that `[<EqualsAssignmentOrSpaced>]` in combination with arity 2 arguments be completely illegal, since functionally it makes no difference from the standard EqualsAssignment attribute. That is to say it should fail with a check in `Precompute.fs`.

Reply:
Good shout - all done.